cli/alert_add.go

```diff
@@ -74,7 +75,7 @@ func (a *alertAddCmd) addAlert(ctx context.Context, _ *kingpin.ParseContext) err
 		// Allow the alertname label to be defined implicitly as the first argument rather
 		// than explicitly as a key=value pair.
 		if _, err := parseLabels([]string{a.labels[0]}); err != nil {
-			a.labels[0] = fmt.Sprintf("alertname=%s", a.labels[0])
+			a.labels[0] = fmt.Sprintf("alertname=%s", strconv.Quote(a.labels[0]))
```

Comment:
This is quoted to make sure commands such as `amtool alert add "foo with spaces"` is parsed as `alertname="foo with spaces"` and not `alertname=foo with spaces` - the latter being invalid in the UTF-8 parser.